src/ts/view/View.ts

```diff
@@ -0,0 +1,90 @@
+import { $, $$ } from '../utils';
+import { PATH_ID, STORAGE_ID } from '../constants';
+import { VendingMachineInterface } from '../domain/VendingMachine';
+import ProductManageView, { ProductManageViewInterface } from './ProductManageView';
+import RechargeView, { RechargeViewInterface } from './RechargeView';
+
+export default class View {
+  $$tabResultContainers: NodeListOf<HTMLTableSectionElement>;
+  $tabProductManageButton: HTMLInputElement;
+  $tabRechargeButton: HTMLInputElement;
+  $tabPurchaseProductButton: HTMLInputElement;
+  $$tabButtons: NodeListOf<HTMLInputElement>;
+  vendingMachine: VendingMachineInterface;
+  productManageView: ProductManageViewInterface;
+  rechargeView: RechargeViewInterface;
+  currentTab: string;
+
+  constructor(vendingMachine: VendingMachineInterface) {
+    this.$$tabResultContainers = <NodeListOf<HTMLTableSectionElement>>$$('.tab-result-container');
+    this.$tabProductManageButton = <HTMLInputElement>$('#tab-product-manage');
+    this.$tabRechargeButton = <HTMLInputElement>$('#tab-recharge');
+    this.$tabPurchaseProductButton = <HTMLInputElement>$('#tab-purchase-product');
+    this.$$tabButtons = <NodeListOf<HTMLInputElement>>$$('.tab-button');
+    this.vendingMachine = vendingMachine;
```

Comment:
```suggestion
    this.$$tabButtons = <NodeListOf<HTMLInputElement>>$$('.tab-button');
    
    this.vendingMachine = vendingMachine;
```
성격이 다른 두 초기화 코드 중간에 한번 개행해주시면 가독성에 더 좋겠어요~

Reply:
코드 사이에 적절한 개행을 통해 가독성을 좋게 하도록 하겠습니다! ㅎㅎ